src/ripple/net/RPCCall.h

```diff
@@ -65,18 +65,22 @@ fromNetwork(
     std::unordered_map<std::string, std::string> headers = {});
 }  // namespace RPCCall
 
-/** Given a rippled command line, return the corresponding JSON.
- */
 Json::Value
-cmdLineToJSONRPC(std::vector<std::string> const& args, beast::Journal j);
+rpcCmdToJson(
+    std::vector<std::string> const& args,
+    Json::Value& retParams,
+    beast::Journal j,
+    unsigned int apiVersion);
```

Comment:
Maybe not for this PR, but we should consider making apiVersion either an enum type with one enum per API version or maybe a strongly-typed int (int with a tag, like we do with base_uint).